src/sass/modules/_m-schemaform.scss

```diff
@@ -284,7 +284,7 @@ label {
   }
   > .schemaform-file-item-edit {
     flex-direction: column;
-    align-items: left;
```

Comment:
Should we automate this check somehow (e.g. [doiuse](https://github.com/anandthakker/doiuse))?